Bundle/PageBundle/Helper/PageHelper.php

```diff
@@ -199,7 +199,7 @@ public function renderPageByUrl($uri, $url, $locale, $layout = null)
                 if ($result instanceof Redirection) {
                     return new RedirectResponse($this->container->get('victoire_widget.twig.link_extension')->victoireLinkUrl(
                         $result->getLink()->getParameters()
-                    ));
+                    ), 301);
```

Comment:
We can use a constant here:

1. import the class: `use Symfony\Component\HttpFoundation\Response;`
2. use `Response::HTTP_MOVED_PERMANENTLY` instead of the number `301`

It helps reading because it shows a readable text instead of a obscure number.

Reply:
and then please do the same PR to 2.3 branch